frontend/components/LiveQuery/SelectTargets.tsx

```diff
@@ -263,12 +262,6 @@ const SelectTargets = ({
     labelsSummary && setLabels(parseLabels(labelsSummary));
   }, [labelsSummary]);
 
-  useEffect(() => {
-    if (inputTabIndex === null && labelsSummary && teams) {
-      setInputTabIndex(labelsSummary.length + teams.length || 0);
```

Comment:
This is incorrect as it counts all labels including hidden ones. It works without this state.